deisctl/deisctl.go

```diff
@@ -41,6 +41,9 @@ Commands, use "deisctl help <command>" to learn more:
   refresh-units     refresh unit files from GitHub
   ssh               open an interacive shell on a machine in the cluster
   help              show the help screen for a command
+  upgrade-prep      prepare a running cluster for upgrade
+  upgrade-takeover  allow an upgrade to gracefully takeover a running cluster
+  rolling-restart   perform a rolling restart for instance units
```

Comment:
What's the meaning of "instance unit" here? The term is unfamiliar to me in this context.